guides/source/active_record_connection_management.md

```diff
@@ -0,0 +1,133 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON http://guides.rubyonrails.org.**
+
+Active Record Connection Management
+===================================
+
+In general, Active Record will automagically manage database connections in an
+efficient manner. See [Configuring a Database](/configuring.html#configuring-a-database)
+and [Database pooling](configuring.html#database-pooling) for information about the
+basics of optimally configuring your application to use available resources.
+
+Sometimes, you will need to manage Active Record connections and/or connection
+pools yourself. These situations are:
+
+* Configuring a multi-process web server
+* Spawning processes
+
+In previous version of rails, you needed to think about managing connections when
+spawning threads, but this is no longer the case.
+
+--------------------------------------------------------------------------------
+
+Configuring a multi-process web server
+--------------------------------------
+
+Each process in a multi-process web server needs its own database connection pool.
+If the server is also multi-threaded, the threads within each process will automatically
+share that processes' connection pool.
```

Comment:
I believe `processes'` is plural, and should be changed to either `process's` or `process'`.